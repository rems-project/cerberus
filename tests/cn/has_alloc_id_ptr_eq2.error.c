int f(int *p, int *q)
/*@
requires
    has_alloc_id(p);
    has_alloc_id(q);
ensures
    return == 0i32;
@*/
{
    return p == q;
}

