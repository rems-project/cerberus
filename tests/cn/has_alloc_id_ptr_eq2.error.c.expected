return code: 1
[1/1]: f -- fail
tests/cn/has_alloc_id_ptr_eq2.error.c:10:5: error: Unprovable constraint
    return p == q;
    ^~~~~~~~~~~~~~ 
Constraint from tests/cn/has_alloc_id_ptr_eq2.error.c:7:5:
    return == 0i32;
    ^~~~~~~~~~~~~~~ 
State file: file:///tmp/state__has_alloc_id_ptr_eq2.error.c__f.html
